pkg/jx/cmd/cmd.go

```diff
@@ -285,11 +283,21 @@ func fullPath(command *cobra.Command) string {
 }
 
 func setLoggingLevel(cmd *cobra.Command, args []string) {
-	level := cmd.Flag(opts.OptionLogLevel).Value.String()
-	err := log.SetLevel(level)
+	verbose, err := strconv.ParseBool(cmd.Flag(opts.OptionVerbose).Value.String())
```

Comment:
What's the point really of `--verbose` at this stage. Is it not really enough with `--log-level`. All --verbose is doing is changing the level to debug.

If we are going down this route, I'd argue we should get rid of `--verbose` altogether.

Reply:
What was the intention of --verbose in the first place? I guess it was adding additional info level messages to the user. Is this really useful? Not sure.

Reply:
so, it looks like --log-level is only implemented in a handful of commands, where as --verbose is implemented nearly everywhere (its also all over our docs site and in all our demos). This seemed  like the change that would have the least impact to the users.

Reply:
So are you suggesting to remove `--log-level` and just use `--verbose`? That would imply we only have info and debug level to work with, right? Seems a bit limiting. 

If we cannot get rid of `verbose` maybe we need to keep both where `--verbose` is effectively some sort of alias for `--log-level=debug`. Also `log-level` taking precedence over `verbose`.

Another alternative could be to just keep `verbose` as a CLI options, but allow to set any type of log level via an env variable, eg `JX_LOG_LEVEL`.

Reply:
I like the idea of `Another alternative could be to just keep verbose as a CLI options, but allow to set any type of log level via an env variable, eg JX_LOG_LEVEL.` . I think that would work and have the least impact on existing users

Reply:
I think there are also different type of users. For jx CLI users for which the differentiation between verbose [true|false] is probably enough. 

For developers, however, trace level logging might be useful (eg thinking about logging in a loop)